ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/KsqlResourceTest.java

```diff
@@ -288,8 +281,14 @@ public void setUp() throws IOException, RestClientException {
 
     streamName = KsqlIdentifierTestUtil.uniqueIdentifierName();
 
-    when(schemaInjectorFactory.apply(any())).thenReturn(sandboxSchemaInjector);
-    when(schemaInjectorFactory.apply(serviceContext)).thenReturn(schemaInjector);
+    when(schemaInjectorFactory.apply(any())).thenAnswer(inv -> {
```

Comment:
I think it would be clearer to use mockito hamcrest here:
```
when(schemaInjectorFactory.apply(argThat(isIn(serviceContexts)))).thenReturn(schemaInjector);	
when(schemaInjectorFactory.apply(any())).thenReturn(sandboxSchemaInjector);	
```

Reply:
Nice. I like that proposal better.